freqtrade/loggers/json_formatter.py

```diff
@@ -0,0 +1,20 @@
+import json
+import logging
+import time
+
+
+class JsonFormatter(logging.Formatter):
+    """
+    Custom logging formatter that outputs logs in structured JSON format.
+    """
+
+    def format(self, record):
+        log_entry = {
+            "timestamp": time.strftime("%Y-%m-%dT%H:%M:%S", time.gmtime(record.created)),
```

Comment:
i'm ... not sure why we need to invent a new time format?

I'd prefer to use ISO format intead - which is slightly different to what you're printing here.